### DISCUSSION TITLE
404, and password authentication failed

### DISCUSSION THREAD

**Discussion by mymmbj** [teslamate-org/teslamate#1588] [Getting started]
**Describe the bug**
404 page not found 
pq: password authentication failed for user "teslamate

**Expected behavior**
can save and test the password, and log in tesla account

**How to reproduce it (as minimally and precisely as possible)**:

```
paste logs here
```
i used the defalut setting from the doc.
and when i logged in https://my-website, enter my tesla account and password, it always return '404 page not found'.

but still, i am able to access https://my-website/grafana. it's like issue #875 (#875)

than i go to --configuration--databast--teslamate(page https://my-website/grafana/datasources/edit/1/), i enter the password 'secret' (which i use the default code) and press save&test , it pops up that 'pq: password authentication failed for user "teslamate"'.
![image](https://user-images.githubusercontent.com/83386054/116503479-c8d1be00-a8e8-11eb-97b6-aaf02d496e92.png)
![image](https://user-images.githubusercontent.com/83386054/116503485-ccfddb80-a8e8-11eb-9d27-edc00501497e.png)
in the file .htpasswd, i use the encrypt password teslamate:$2y$10$f7PB3UF3PNzqMIXZmf1dIefOkrv/15Xt6Xw3pzc6mkS/B5qoWBdAG
(different from issue #875 )

What i think
not being able to access to the database, causes the '404 page not found'.

What i'm asking for help
how to solve the 'pq: password authentication failed for user "teslamate"' problem.

it's my first time here, if i'm offending any rules here,please let me know, my apologies and my thanks.

**Screenshots**

**Data**

**Environment**

- TeslaMate version: 
- Type of installation: Docker 
- OS TeslaMate is installed on: centos 7
- User OS & Browser:  Mobaxterm, chrome
- Others:

**Comment by mymmbj** [comment]
i used codes in this page:
https://docs.teslamate.org/docs/guides/traefik

just modified my website address and nothing more.

**Comment by goodtiger** [comment]
I have got the same issue.
